### DISCUSSION TITLE
How do I find a path inside an element when defining them, or when clicking on an object without focusing on it?

### DISCUSSION THREAD

**Discussion by Rndjester** [ianstormtaylor/slate#5770] [Q&A]
![image](https://github.com/user-attachments/assets/3bcf9098-198e-4922-9bb1-ad3564f80487)

**Comment by WindRunnerMax** [comment]
You just need to use ReactEditor.findPath(editor, element). However, keep in mind that you must use it after the node has rendered because it can change.
